Tests/IntegrationTests/e2e-docker.sh

```diff
@@ -77,8 +79,11 @@ for fixture in $(pwd)/Tests/IntegrationTests/Fixtures/*/; do
 
         # TODO: optimize this
         cd TestDistribution
+        composer reinstall neos/test-nodetypes
+        composer reinstall neos/test-site
         ./flow flow:package:rescan > /dev/null
         ./flow flow:cache:flush > /dev/null
+
```

Comment:
this change is different to https://github.com/neos/neos-ui/commit/5d9d09ce8ad7cb0c987a2aaf00d9f7ef76c71ae9
is there a reason for this?

(and will result in a merge conflict)

Reply:
I wasn't sure why Markus did the change like he did and he didn't explain in his PR. So I just took the lines that were necessary to get it running again. 
@markusguenther do you still remember?

I mean, why cache warmup when you run a command afterwards anyway and why show the config, etc.

Reply:
We had issues in the past with circle ci and then the tests failed on the package change. Just used the changes from the CI pipeline tests.

Reply:
im fine with merging as is, but when upmerging it we should ignore this change and keep 8.2 different?

Reply:
I can also adjust it to do the same. Fine for me.